packages/service-library/src/servicelib/monitoring.py

```diff
@@ -101,3 +101,5 @@ def setup_monitoring(app: web.Application, app_name: str):
 
     # Checks that middleware is in the outermost layer
     app.on_startup.append(check_outermost_middleware)
+
+    return True
```

Comment:
missing return type. really necessary ?there is no other path...

Reply:
Actually with the new decorator, you do not need to return anything and it is interpreted as OK